fabric-loot-tables-v1/src/main/java/net/fabricmc/fabric/api/loot/v1/visitor/LootTableVisitor.java

```diff
@@ -0,0 +1,65 @@
+package net.fabricmc.fabric.api.loot.v1.visitor;
+
+public interface LootTableVisitor {
```

Comment:
I think this needs some `default`s 😄

Reply:
Yes, definitely, once I add methods to nodes that allow getting their children, which will come once I add the dozens of different nodes required for all vanilla functions, loot conditions, and whatever else I deem necessary.